src/raftstore/store/peer.rs

```diff
@@ -444,13 +447,36 @@ impl Peer {
             ready.hs.take();
         }
 
+        if let Some(ref soft_state) = ready.ss {
```

Comment:
When we receive a message, we will create a replicated peer, and if this peer doesn't receive any  message later, the peer can be considered stale too. 
But this peer can't be ready, so I think we should check this in `tick` function.